.github/workflows/release.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           node-version: 18.x
 
       - name: Install Dependencies
-        run: yarn
+        run: yarn install --no-immutable
```

Comment:
Just happened to notice this. Yarn does a decent job recognizing it's in CI and then enabling this flag by default but better safe than sorry when it comes to releases